Sources/Basic/Lock.swift

```diff
@@ -36,7 +36,11 @@ enum ProcessLockError: Swift.Error {
 /// by mutiple instances of a process. The `FileLock` is not thread-safe.
 public final class FileLock {
     /// File descriptor to the lock file.
+  #if os(Windows)
+    private var h: HANDLE?
```

Comment:
Why not at least `handle` instead of `h` for this variable?

Reply:
Renaming to `handle` would be nice to have. Feel free to make a PR :)

Reply:
@aciidb0mb3r PR #2093 ;)